### ISSUE TITLE
Issues with installing Archivematica 1.8 RPMs/Debs on Fresh Servers

### ISSUE THREAD

**Issue by jpellman** [artefactual/archivematica#810]
Using the documentation located [here](https://www.archivematica.org/en/docs/archivematica-1.8/admin-manual/installation/installation/#installation), I encountered the following issues with the CentOS and Ubuntu packages posted to last meeting's agenda:

## CentOS 7

At step 3 of the instructions in the documentation, I got the following error:

```
[centos@ip-172-31-8-206 ~]$ sudo -u root yum install -y java-1.8.0-openjdk-headless mariadb-server gearmand
Loaded plugins: fastestmirror
https://jenkins-ci.archivematica.org/1.8.x/centos/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to jenkins-ci.archivematica.org:443; Connection refused"
Trying other mirror.
https://jenkins-ci.archivematica.org/1.8.x/centos/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to jenkins-ci.archivematica.org:443; Connection refused"
Trying other mirror.
https://jenkins-ci.archivematica.org/1.8.x/centos/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to jenkins-ci.archivematica.org:443; Connection refused"
Trying other mirror.
https://jenkins-ci.archivematica.org/1.8.x/centos/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to jenkins-ci.archivematica.org:443; Connection refused"
Trying other mirror.
https://jenkins-ci.archivematica.org/1.8.x/centos/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to jenkins-ci.archivematica.org:443; Connection refused"
Trying other mirror.
https://jenkins-ci.archivematica.org/1.8.x/centos/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to jenkins-ci.archivematica.org:443; Connection refused"
Trying other mirror.
https://jenkins-ci.archivematica.org/1.8.x/centos/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to jenkins-ci.archivematica.org:443; Connection refused"
```

This was just due to a difference in the URL in the documentation vs the URL from last meeting. I updated the yum repo to use the correct URL and received the following:

```
[centos@ip-172-31-8-206 ~]$ sudo -u root yum install -y python-pip archivematica-storage-service
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirrors.usc.edu
 * epel: mirrors.kernel.org
 * extras: mirror.web-ster.com
 * updates: mirror.keystealth.org
No package archivematica-storage-service available.
```

So I was unable to proceed past step 3 in the instructions since the storage service wasn't available.  I did try the steps afterwards just to see how far I could get without the storage service and got up to the following (step 5):

```
[centos@ip-172-31-8-206 ~]$ sudo -u archivematica bash -c " \
> set -a -e -x
> source /etc/sysconfig/archivematica-dashboard
> cd /usr/share/archivematica/dashboard
> /usr/lib/python2.7/archivematica/dashboard/bin/python manage.py syncdb --noinput
> ";
+ source /etc/sysconfig/archivematica-dashboard
++ ARCHIVEMATICA_DASHBOARD_DASHBOARD_DJANGO_SECRET_KEY=Ptpucrhu0doIq2QcHZtcO9caaqE11fk2
++ ARCHIVEMATICA_DASHBOARD_DASHBOARD_DJANGO_ALLOWED_HOSTS='*'
++ AM_GUNICORN_BIND=127.0.0.1:7400
++ DJANGO_SETTINGS_MODULE=settings.production
++ ARCHIVEMATICA_DASHBOARD_DB_NAME=MCP
++ ARCHIVEMATICA_DASHBOARD_DB_USER=archivematica
++ ARCHIVEMATICA_DASHBOARD_DB_PASSWORD=demo
++ ARCHIVEMATICA_DASHBOARD_DB_HOST=localhost
++ ARCHIVEMATICA_DASHBOARD_DB_PORT=3306
++ ARCHIVEMATICA_DASHBOARD_GEARMAN=localhost:4730
++ ARCHIVEMATICA_DASHBOARD_ELASTICSEARCH=localhost:9200
++ PYTHONPATH=/usr/lib/archivematica/archivematicaCommon/:/usr/share/archivematica/dashboard
+ cd /usr/share/archivematica/dashboard
+ /usr/lib/python2.7/archivematica/dashboard/bin/python manage.py syncdb --noinput
bash: line 3: /usr/lib/python2.7/archivematica/dashboard/bin/python: No such file or directory
```

## Ubuntu 16.04

I got up to step 3 on Ubuntu.  When I ran `apt-get update` I received the following:

```
Reading package lists... Done
W: The repository 'http://jenkins-ci.archivematica.org/1.8.x/ubuntu xenial Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: http://packages.archivematica.org/1.6.x/ubuntu-externals/dists/trusty/InRelease: Signature by key 486650CDD6355E25DA542E06C8F04D025236CA08 uses weak digest algorithm (SHA1)
E: Failed to fetch http://jenkins-ci.archivematica.org/1.8.x/ubuntu/dists/xenial/main/binary-amd64/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

Again there was a slight gap between the URLs from our last meeting and the documentation.  When I switched the URL to use what was in our meeting notes I was met with:

```
Reading package lists... Done
W: The repository 'http://jenkins-ci.archivematica.org/repos/apt/dev-1.8.x-xenial xenial Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: http://packages.archivematica.org/1.6.x/ubuntu-externals/dists/trusty/InRelease: Signature by key 486650CDD6355E25DA542E06C8F04D025236CA08 uses weak digest algorithm (SHA1)
E: Failed to fetch http://jenkins-ci.archivematica.org/repos/apt/dev-1.8.x-xenial/dists/xenial/main/binary-amd64/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
```
This looks like it might just be due to how the repo is structured, since the current release has `dist` and architecture-specific subdirs.

**Comment by scollazo** [comment]
Those are the scripts I'm using for testing:

https://gist.github.com/scollazo/4bd044ced115a09370274e157cc5536a#file-centos-1-8-sh
https://gist.github.com/scollazo/4bd044ced115a09370274e157cc5536a#file-xenial-1-8-sh

Some urls/paths have changed, and those changes are not reflected on the docs for now. I'll review the docs to sync them with this scripts.

**Comment by jpellman** [comment]
Tried a fresh install on a new batch of EC2 instances.  Here are my results:

## CentOS 7

Installation went through without any obvious issues.  In the post-install instructions, however, step 1 gives the wrong port number (8000 instead of 8001) and step 3 doesn't give the dashboard port (81).

## Ubuntu 16.04

At step 8 in the instructions, the bit where you're supposed to give the archivematica user a password of "demo" is a bit confusing, since the package script dialog doesn't prompt you to create a password for a user "archivematica," but rather emphasizes "archivematica-mcp-server":

```
                                              ┌─────────────────────────────────────────────────────┤ Configuring archivematica-mcp-server ├──────────────────────────────────────────────────────┐
                                              │ Please provide a password for archivematica-mcp-server to register with the database server. If left blank, a random password will be generated.  │
                                              │                                                                                                                                                   │
                                              │ MySQL application password for archivematica-mcp-server:                                                                                          │
                                              │                                                                                                                                                   │
                                              │ ****_____________________________________________________________________________________________________________________________________________ │
                                              │                                                                                                                                                   │
                                              │                                            <Ok>                                                <Cancel>                                           │
                                              │                                                                                                                                                   │
                                              └───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

At step 11, the fits service fails to start and the restart command for gearman fails (missing a 'service').

```
ubuntu@ip-172-31-11-52:~$ sudo service fits start
Failed to start fits.service: Unit fits.service not found.
ubuntu@ip-172-31-11-52:~$ sudo restart gearman-job-server
sudo: restart: command not found
```

In the post-install portion of the instructions, the port number is inconsistent between CentOS 7 and Ubuntu 16.04. 8001 doesn't work for Ubuntu and 8000 won't work for CentOS.  I also noticed that the CentOS instructions tell you which ports these services are listening on immediately after the step that you perform to install them while the Ubuntu instructions don't, and it might be nice to make the instructions more consistent between platforms on that front.

Finally, I noticed that the nginx config for the dashboard that you create a symlink to in step 9 doesn't actually exist, so it's not possible to get to the dashboard in step 3 of the post-install.

**Comment by scollazo** [comment]
The problems with xenial install are very strange, I can't reproduce them. 

Can you post the output of "dpkg -l | grep archivematica"?

**Comment by mamedin** [comment]
Hi jpellman,

I agree that the ports used by the dashboard and the Storage Service should be clarified at the document.

As you well say, also the command to restart gearmand in case of problems should be changed in ubuntu 16 to the systemd version.

But, I can't reproduce the nginx dashboard config file and startup of the fits service issues. Can you check that the `archivematica-dashboard` and` fits` packages have been installed? 

Please add to what scollazo has asked you, the output of "dpkg -l | grep fits"

**Comment by jpellman** [comment]
Hi @scollazo and @mamedin ,

Here is the output of the two dpkg commands:

```
ubuntu@ip-172-31-1-44:~$ dpkg -l | grep archivematica
ii  archivematica-common                 1:1.8.0~columbia12                         amd64        Common libraries for archivematica
ii  archivematica-mcp-server             1:1.8.0~columbia12                         amd64        MCP Server for Archivematica
ii  archivematica-storage-service        1:0.11.0~rc.2-1-gd230939                   amd64        Django webapp for managing storage in an Archivematica
ubuntu@ip-172-31-1-44:~$ dpkg -l | grep fits
ubuntu@ip-172-31-1-44:~$
```

The issue seems to be due to me copying and pasting the code from the instructions directly into terminal.  The first command was executed / databases were set up / etc but the second two apt commands weren't executed (possibly due to the 2 MySQL setup screens / being piped elsewhere).  If I copy and paste each of those commands individually it seems to work.  

I also tried consolidating all the apt commands onto one line with:

```
sudo apt-get install -y archivematica-mcp-server archivematica-dashboard archivematica-mcp-client
```

But that didn't seem to want to work either and gave me the same output from dpkg (I ran this after I had already pasted in the code from the install instructions though and archivematica-mcp-server was already installed, so maybe that could have affected this).

I'm going to check and see if I can get through the steps after step 8 now.

**Comment by jpellman** [comment]
I was now able to get up to step 3 of the post-install instructions in Ubuntu.  When I tried to create a new user and log in on the dashboard, however, I was presented with an internal server error.  The following was logged to /var/log/archivematica/dashboard/dashboard.log when that happened:

```
WARNING   2017-12-01 19:44:28  py.warnings:loaddata:find_fixtures:239:  /usr/local/lib/python2.7/dist-packages/django/core/management/commands/loaddata.py:239: RemovedInDjango19Warning: initial_data fixtures are deprecated. Use data migrations instead.
  RemovedInDjango19Warning

ERROR     2017-12-01 20:01:46  django.request:base:handle_uncaught_exception:256:  Internal Server Error: /installer/welcome/
Traceback (most recent call last):
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/share/archivematica/dashboard/installer/views.py", line 51, in welcome
    api_key = ApiKey.objects.create(user=user)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/query.py", line 348, in create
    obj.save(force_insert=True, using=self.db)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/tastypie/models.py", line 46, in save
    return super(ApiKey, self).save(*args, **kwargs)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/base.py", line 734, in save
    force_update=force_update, update_fields=update_fields)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/base.py", line 762, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/base.py", line 846, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/base.py", line 885, in _do_insert
    using=using, raw=raw)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/query.py", line 920, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 974, in execute_sql
    cursor.execute(sql, params)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/django/db/backends/mysql/base.py", line 124, in execute
    return self.cursor.execute(query, args)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 226, in execute
    self.errorhandler(self, exc, value)
  File "/usr/share/python/archivematica-dashboard/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'MCP.tastypie_apikey' doesn't exist")
```

**Comment by scollazo** [comment]
Seems like the dashboard database has not been populated. 

Can you try with "dpkg-reconfigure archivematica-dashboard"?

**Comment by jpellman** [comment]
Hi @scollazo,

I ran `dpkg-reconfigure archivematica-dashboard` and looked through the output and then came across this:

```
  File "/usr/share/archivematica/dashboard/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/migrate.py", line 222, in handle
    executor.migrate(targets, plan, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/executor.py", line 110, in migrate
    self.apply_migration(states[migration], migration, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/executor.py", line 148, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/migration.py", line 112, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/operations/special.py", line 183, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/usr/local/lib/python2.7/dist-packages/fpr/migrations/0015_pronom_92.py", line 12, in data_migration
    call_command('loaddata', fixture_file, app_label='fpr')
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 120, in call_command
    return command.execute(*args, **defaults)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/loaddata.py", line 60, in handle
    self.loaddata(fixture_labels)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/loaddata.py", line 100, in loaddata
    self.load_label(fixture_label)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/loaddata.py", line 157, in load_label
    obj.save(using=self.using)
  File "/usr/local/lib/python2.7/dist-packages/django/core/serializers/base.py", line 173, in save
    models.Model.save_base(self.object, using=using, raw=True)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/base.py", line 762, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/base.py", line 846, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/base.py", line 885, in _do_insert
    using=using, raw=raw)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 920, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/compiler.py", line 974, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/mysql/base.py", line 124, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python2.7/dist-packages/MySQLdb/cursors.py", line 226, in execute
    self.errorhandler(self, exc, value)
  File "/usr/local/lib/python2.7/dist-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorvalue
django.db.utils.DataError: Problem installing fixture '/usr/local/lib/python2.7/dist-packages/fpr/migrations/pronom_92.json': Could not load fpr.Format(pk=None): (1406, "Data too long for column 'slug' at row 1")
```

I also came across this when I ran `apt-get update` in step 3 of the Ubuntu 16.04 instructions (although I don't believe it affected later steps):

```
Reading package lists... Done
W: http://jenkins-ci.archivematica.org/repos/apt/dev-1.8.x-xenial/./Release.gpg: Signature by key D61AF7877704501266B2C0626576A0E00F4A4D31 uses weak digest algorithm (SHA1)
W: Invalid 'Date' entry in Release file /var/lib/apt/lists/partial/jenkins-ci.archivematica.org_repos_apt_dev-1.8.x-xenial_._Release
W: http://jenkins-ci.archivematica.org/repos/apt/release-0.11-xenial/./Release.gpg: Signature by key D61AF7877704501266B2C0626576A0E00F4A4D31 uses weak digest algorithm (SHA1)
W: Invalid 'Date' entry in Release file /var/lib/apt/lists/partial/jenkins-ci.archivematica.org_repos_apt_release-0.11-xenial_._Release
```

**Comment by scollazo** [comment]
This comes from https://github.com/artefactual/archivematica/issues/792

New packages up to version ~columbia13 have the fix for this issue, but you'll need to recreate the database.

So, in resume:
<pre>
sudo apt-get update
sudo apt-get upgrade
</pre>

This should bring the updated packages into the system.

And then, recreate the database with:
<pre>
dpkg-reconfigure archivematica-mcp-server (choose drop and create the db)
dpkg-reconfigure archivematica-dashboard
</pre>

**Comment by jpellman** [comment]
Hi @scollazo ,

I tried doing a fresh install, but I'm still running into the issue with the slug field.  Is it possible that I was using the wrong repo or the updated package hasn't bee uploaded to the repo used in the docs yet?

**Comment by jpellman** [comment]
(As an aside, I've never actually seen the "NifTI" acronym expanded before- it took a second before I recognized it in its full form.)

**Comment by scollazo** [comment]
Hi @jpellman , there was a bug indeed with ubuntu xenial packages, that has been fixed in artefactual-labs/am-packbuild@3c654c5ceade68b545858d7bc5ca7394bb7bffa3

You should be able to proceed now with the xenial install without issues. I recorded this asciicast in case you have doubts with some step:

https://asciinema.org/a/UGniNfHlWHHW0A5QcPYrlCgpy

**Comment by jpellman** [comment]
Hi @scollazo ,

The installation for Ubuntu works now and nginx does not experience an internal server error.  However, I've noticed that when I rebooted the Ubuntu VM I was testing on fits did not start up:

```
ubuntu@ip-172-31-9-119:~$ sudo service fits status
● fits.service - FITS Nailgun server
   Loaded: loaded (/lib/systemd/system/fits.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
```

I tried rebooting a CentOS VM after installation and was unable to replicate the issue.  I believe this might be do to a difference in the version of fits-nailgun used between the two packages (nailgun-server-debian.jar iirc for Ubuntu and nailgun-server-0.9.2-SNAPSHOT.jar for CentOS 7).

---

Another issue that I ran into occurred on a CentOS VM as well.  At one point during an installation, I accidentally misentered the storage service URL in the dashboard.  This produced the following error in the web interface:

```
Storage server inaccessible. Please contact an administrator or update storage service URL below.
```

I attempted to update the storage service URL without luck.  The contents of /var/log/archivematica/dashboard/dashboard.log displayed a number of errors similar to the following:

```
HTTPError: 404 Client Error: NOT FOUND for url: http://localhost:8001/api/v2/pipeline/823c3af0-7940-46cb-8ec2-87a8759566fc/
ERROR     2017-12-05 18:39:15  archivematica.dashboard:views:transfer_source_locations:59:  Error retrieving source directories
Traceback (most recent call last):
  File "/usr/share/archivematica/dashboard/components/transfer/views.py", line 56, in transfer_source_locations
    return helpers.json_response(storage_service.get_location(purpose="TS"))
  File "/usr/lib/archivematica/archivematicaCommon/storageService.py", line 149, in get_location
    pipeline = _get_pipeline(get_setting('dashboard_uuid'))
  File "/usr/lib/archivematica/archivematicaCommon/storageService.py", line 123, in _get_pipeline
    response.raise_for_status()
  File "/usr/share/python/archivematica-dashboard/lib/python2.7/site-packages/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
HTTPError: 404 Client Error: NOT FOUND for url: http://localhost:8001/api/v2/pipeline/823c3af0-7940-46cb-8ec2-87a8759566fc/
WARNING   2017-12-05 18:39:21  archivematica.common:storageService:_get_pipeline:122:  This Archivematica instance is not registered with the storage service or has been disabled.
WARNING   2017-12-05 18:39:21  archivematica.common:storageService:_get_pipeline:125:  Error fetching pipeline
Traceback (most recent call last):
  File "/usr/lib/archivematica/archivematicaCommon/storageService.py", line 123, in _get_pipeline
    response.raise_for_status()
  File "/usr/share/python/archivematica-dashboard/lib/python2.7/site-packages/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
HTTPError: 404 Client Error: NOT FOUND for url: http://localhost:8001/api/v2/pipeline/823c3af0-7940-46cb-8ec2-87a8759566fc/
WARNING   2017-12-05 18:40:24  archivematica.common:storageService:_get_pipeline:122:  This Archivematica instance is not registered with the storage service or has been disabled.
WARNING   2017-12-05 18:40:24  archivematica.common:storageService:_get_pipeline:125:  Error fetching pipeline
Traceback (most recent call last):
  File "/usr/lib/archivematica/archivematicaCommon/storageService.py", line 123, in _get_pipeline
    response.raise_for_status()
  File "/usr/share/python/archivematica-dashboard/lib/python2.7/site-packages/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
HTTPError: 404 Client Error: NOT FOUND for url: http://127.0.0.1:8001/api/v2/pipeline/823c3af0-7940-46cb-8ec2-87a8759566fc/
WARNING   2017-12-05 18:40:30  archivematica.common:storageService:_get_pipeline:122:  This Archivematica instance is not registered with the storage service or has been disabled.
WARNING   2017-12-05 18:40:30  archivematica.common:storageService:_get_pipeline:125:  Error fetching pipeline
Traceback (most recent call last):
  File "/usr/lib/archivematica/archivematicaCommon/storageService.py", line 123, in _get_pipeline
    response.raise_for_status()
  File "/usr/share/python/archivematica-dashboard/lib/python2.7/site-packages/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
HTTPError: 404 Client Error: NOT FOUND for url: http://127.0.0.1:8001/api/v2/pipeline/823c3af0-7940-46cb-8ec2-87a8759566fc/
WARNING   2017-12-05 18:42:10  archivematica.common:storageService:_get_pipeline:122:  This Archivematica instance is not registered with the storage service or has been disabled.
WARNING   2017-12-05 18:42:10  archivematica.common:storageService:_get_pipeline:125:  Error fetching pipeline
Traceback (most recent call last):
  File "/usr/lib/archivematica/archivematicaCommon/storageService.py", line 123, in _get_pipeline
    response.raise_for_status()
  File "/usr/share/python/archivematica-dashboard/lib/python2.7/site-packages/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
HTTPError: 404 Client Error: NOT FOUND for url: http://127.0.0.1:8001/api/v2/pipeline/823c3af0-7940-46cb-8ec2-87a8759566fc/
```

**Comment by scollazo** [comment]
I'll try to recreate the issue with fits, and to recreate the one in centos.

**Comment by kellyannewithane** [comment]
@scollazo @jhsimpson is this issue closed?

**Comment by jpellman** [comment]
Since there were two separate issues in my last comment, I've separated out the fits issue into #868 for further tracking as per last meeting's CUL action item.

**Comment by kellyannewithane** [comment]
Closing this issue because SS URL issue is addressed in #861 and the FITS issue is addressed in #868

**Comment by jpellman** [comment]
Hi,

This might have gotten lost amidst all of the other issues here, but (as per an earlier remark in this thread) one of the problems I ran into involved the post-install instructions on CentOS:

> Installation went through without any obvious issues. In the post-install instructions, however, step 1 gives the wrong port number (8000 instead of 8001) and step 3 doesn't give the dashboard port (81).

It'll be easier to track if this is in the docs repo so I'll open a separate issue there.
